server/tracedb/tempodb.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 func tempoDefaultPorts() []string {
-	return []string{"9095"}
+	return []string{"9095", ""}
```

Comment:
For http urls like www.grafana.com/traces the port is "not" setup

Ideally, I'd like us to change this logic to allow each integration read the config file and based on the values decide what is the expected list of port instead of having a static list